components/cluster-secret-store-rh/production/kflux-rhel-p01/trusted-ca-configmap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: trusted-ca
+  namespace: appsre-vault
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
+  labels:
+    config.openshift.io/inject-trusted-cabundle: "true"
+data: {}
+
```

Comment:
we should probably get rid of this field, argo might get upset when the data field gets populated